src/foam/nanos/script/Script.js

```diff
@@ -318,6 +321,7 @@ foam.CLASS({
           setLastDuration(pm.getTime());
           ps.flush();
           setOutput(baos.toString());
+          ((DAO) x.get("scriptEventDAO")).put_(x, this);
```

Comment:
Test.js overrides this .. did you want this to happen at the end of a test too?

Reply:
so does TestRunnerScript, but i can see why we wouldn't save those.

Reply:
Re: save event at end of test - no.  And since it overrides runScript and does not save, I'm good with current implementation.